interpreter.py

```diff
@@ -198,6 +236,10 @@ def exec_operation(self, instruction):
             var_val = self.lookup_var(args[0])
             assert isinstance(var_val, Float)
             return Float(var_val.value)
+        elif opcode == LLVMZExt:
+            var_val = self.lookup_var(args[0])
+            assert isinstance(var_val, Integer)
+            return Integer(var_val.value)
```

Comment:
If `var_val` is a (presumably immutable?) `Integer`, why make a new `Integer` object?